### PR TITLE
[office-js] [office-js-preview] (Outlook) Note item.saveAsync workaro…

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**
…und for Mac

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by ElizabethSamuel-MSFT** [comment]
@davidchesnut Could you take a look? This is related to PRs https://github.com/OfficeDev/office-js-docs-pr/pull/1045 and https://github.com/OfficeDev/outlook-dev-docs/pull/596. Thanks.

**Review by AlexJerabek — CHANGES_REQUESTED** [review]
Two little things:
- The links should be in a `@remarks` section, so Intellisense isn't impacted.
- The locale should be removed from the links if possible.

File: types/office-js-preview/index.d.ts [hunk 1]
```diff
@@ -10748,7 +10748,8 @@ declare namespace Office {
          *
          * **Note**: The following clients have different behavior for saveAsync on appointments in compose mode:
          *
-         * - Mac Outlook does not support saveAsync on a meeting in compose mode. Calling saveAsync on a meeting in Mac Outlook will return an error.
+         * - Outlook for Mac does not support saveAsync on a meeting in compose mode so calling saveAsync in that scenario will return an error.
```
Comment (AlexJerabek) [inline types/office-js-preview/index.d.ts]: Consider splitting this sentence into two for readability.
```suggestion
         * - Outlook for Mac does not support saveAsync on a meeting in compose mode. Calling saveAsync in that scenario returns an error.
```

Comment (AlexJerabek) [inline types/office-js-preview/index.d.ts]: I believe in the last pass you had saveAsync in code ticks. Why the change back?

Comment (ElizabethSamuel-MSFT) [reply types/office-js-preview/index.d.ts]: I removed it for consistency. Nothing else is using ticks in this method and I am not prepared to do such a change right now.

**Review by AlexJerabek — APPROVED** [review]

File: types/office-js-preview/index.d.ts [hunk 2]
```diff
@@ -10785,7 +10786,8 @@ declare namespace Office {
          *
          * **Note**: The following clients have different behavior for saveAsync on appointments in compose mode:
          *
-         * - Mac Outlook does not support saveAsync on a meeting in compose mode. Calling saveAsync on a meeting in Mac Outlook will return an error.
+         * - Outlook for Mac does not support saveAsync on a meeting in compose mode. As such, calling saveAsync in that scenario returns an error.
```
Comment (davidchesnut) [inline types/office-js-preview/index.d.ts]: I recommend using this text from another PR you created. I like this wording better. I think it reads easier.

Outlook for Mac does not support saving a meeting. The `saveAsync` method fails when called from a meeting in compose mode. See [Cannot save a meeting as a draft in Outlook for Mac by using Office JS API](https://support.microsoft.com/help/4505745) for a workaround.

Comment (davidchesnut) [reply types/office-js-preview/index.d.ts]: I'd recommend that text throughout all of these files.
